conda-build/README.md

```diff
@@ -1,9 +1,9 @@
 ### Building for macOS
 
-Install conda packages as decribed here ```conda-build/common/install_conda.sh``` and run ```conda create --name py37 python=3.7 -y;conda activate py37```.
-Running ```python macos_matrix_builder.py``` with python >= 3.7 will start the build process by setting all the environment variables and making a call to conda build. Make sure the meta.yaml file in conda-build/habitat-sim/ is configured correctly accoridng to https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html
+Install conda packages as described here ```conda-build/common/install_conda.sh``` and run ```conda create --name py39 python=3.9 -y;conda activate py39```.
+Running ```python macos_matrix_builder.py``` with python >= 3.9 will start the build process by setting all the environment variables and making a call to conda build. Make sure the meta.yaml file in conda-build/habitat-sim/ is configured correctly according to https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html
```

Comment:
While we're at it, `macos_matrix_builder.py` no longer exists, we unified on `matrix_builder.py`.